mne/io/meas_info.py

```diff
@@ -2150,6 +2150,9 @@ def anonymize_info(info, daysback=None, keep_his=False, verbose=None):
     # file_id and meas_id
     for key in ('file_id', 'meas_id'):
         value = info.get(key)
+        if (value['secs'], value['usecs']) == DATE_NONE:
```

Comment:
The conditional directly below this is either wrong (no need to check for non-None) or this line can fail (when value is None).

Perhaps the conditional 3 lines below should be modified instead, `if none_meas_date or (value['sec'], value['usec]) == DATE_NONE:`